views/cypress/tests/item-authoring-set-response.spec.js

```diff
@@ -169,5 +170,44 @@ describe('Item Authoring', () => {
             cy.get('[data-testid="save-the-item"]').click();
             cy.wait('@saveItem').its('response.body').its('success').should('eq', true);
         });
+        it('can add single interaction to an item', function () {
```

Comment:
I think better to move your test in single file, because this is already heavy and it's about Response in each interaction, and it fails in deferent places for me because of timeouts
Your task is independent from this test
you can just create new item with only one choice interaction and play with response processing of item